platform/viewer/cypress/integration/common/OHIFCornerstoneHotkeys.spec.js

```diff
@@ -85,6 +85,7 @@ describe('OHIF Cornerstone Hotkeys', () => {
     cy.get('@layoutBtn').click();
     //Select 3 viewports
     cy.get('tbody > :nth-child(1) > :nth-child(3)').click();
+    cy.wait(1000);
```

Comment:
Why the wait? And this feels brittle (above line). We should probably at least add a `data-cy` selector for our viewports.

Reply:
Replaced the `wait` for a more fancier condition that checks if message 'Loading...' is being displayed on the viewports.

`Line 87` is selecting the layout style on that table overlay. This one is selecting [1,3]. I can replace this eventually with `setLayout()`, but for now let's just get this code merged, since this is not directly related to the new Preferences Modal tests.